src/components/BookCard.spec.js

```diff
@@ -0,0 +1,10 @@
+import BookCard from './BookCard'
+import { render, screen } from '@testing-library/react'
+
+describe('BookCard', () => {
+  it('renders a booktitle and an author', () => {
```

Comment:
I would test the render of the image as well.

Reply:
Thank you, i just added the test to check whether the image is rendered